pallets/phala/src/stakepool.rs

```diff
@@ -541,11 +560,101 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Manually assign rewards to pools by miningSwitchOrigin for fixing issue 763
+		///
+		/// Requires:
+		/// 1. The caller is root
+		/// 2. Assigned pool must currently exist
+		/// 3. Reward is positive
+		#[pallet::weight(0)]
+		/// remove in v2
+		pub fn force_assign_reward(
+			origin: OriginFor<T>,
+			reward_arr: Vec<(u64, BalanceOf<T>)>,
+		) -> DispatchResult {
+			// Origin must be MiningSwitchOrigin
+			T::MiningSwitchOrigin::ensure_origin(origin)?;
+			for (pid, reward) in reward_arr {
+				// The assigned pool must exist
+				let mut pool_info = Self::ensure_pool(pid)?;
+				// The reward must be positive
+				if reward <= Zero::zero() {
+					continue;
+				}
+				// Assign reward
+				Self::handle_pool_new_reward(&mut pool_info, reward);
+				StakePools::<T>::insert(&pid, &pool_info);
+			}
+			Ok(())
+		}
+
+		/// Claims pool-owner's pending rewards of the sender and send to the `target`
```

Comment:
Please strictly follow the rust docstring style guide. There are spaces between the paragraphs.

Reply:
done

Reply:
Not fixed yet